### ISSUE TITLE
Emoji's On the Message Box

### ISSUE THREAD

**Issue by MuhammadMullah** [papercups-io/papercups#172]
Hi All,

How about some ability to have the user put some emojis whenever they are typing their message on the message box as a form of expression.

**Comment by reichert621** [comment]
hey @MuhammadMullah! we currently have basic emoji support (see https://github.com/papercups-io/papercups/pull/124)

for example, if you type :), <3, `:tada` you should see 😄 ❤️ 🎉  -- is this what you mean?

**Comment by MuhammadMullah** [comment]
No, I meant as in having like a button beside the send button with an option of different emoji signs. Personally I did not realize its there and also it could be difficult and tiresome for someone to know/write that just to get an emoji

**Comment by MuhammadMullah** [comment]
Like how would I know what I am suppose to type just to get a specific emoji e.g `thumbsup`

**Comment by reichert621** [comment]
ah I see 👍 so maybe we could add something like this? https://github.com/missive/emoji-mart

**Comment by MuhammadMullah** [comment]
yes exactly @reichert621
